docs/docs/50-roadmap.md

```diff
@@ -17,20 +17,26 @@ Project](https://github.com/akuity/kargo/milestones)
 
 ## In Progress
 
-### v0.7.0
+### v0.8.0
 
-__Status:__ In Progress
```

Comment:
Being under and "in progress" heading, I don't think we need to repeat the status here.